README.md

```diff
@@ -10,29 +10,30 @@
 Katib is a Kubernetes Native System for [Hyperparameter Tuning][1] and [Neural Architecture Search][2].
 The system is inspired by [Google vizier][3] and supports multiple ML/DL frameworks (e.g. TensorFlow, MXNet, and PyTorch).
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
-
-- [Name](#name)
-- [Concepts in Katib](#concepts-in-katib)
-  - [Experiment](#experiment)
-  - [Suggestion](#suggestion)
-  - [Trial](#trial)
-  - [Worker Job](#worker-job)
-- [Components in Katib](#components-in-katib)
-- [Getting Started](#getting-started)
-- [Web UI](#web-ui)
-- [API Documentation](#api-documentation)
-- [Installation](#installation)
-  - [TF operator](#tf-operator)
-  - [Pytorch operator](#pytorch-operator)
-  - [Katib](#katib)
-  - [Running examples](#running-examples)
-  - [Cleanups](#cleanups)
-- [CONTRIBUTING](#contributing)
-
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+Table of Contents
+=================
+
+  * [Getting Started](#getting-started)
+  * [Name](#name)
+  * [Concepts in Katib](#concepts-in-katib)
+      * [Experiment](#experiment)
+      * [Suggestion](#suggestion)
+      * [Trial](#trial)
+      * [Worker Job](#worker-job)
+  * [Components in Katib](#components-in-katib)
+  * [Web UI](#web-ui)
+  * [API documentation](#api-documentation)
+  * [Installation](#installation)
+      * [TF operator](#tf-operator)
+      * [Pytorch operator](#pytorch-operator)
+      * [Katib](#katib-1)
```

Comment:
Should we name it `[Katib](#katib)` ?
The link (#katib-1) doesn't work for me.

Reply:
Oh yes, it is a typo.